src/Symfony/Bundle/FrameworkBundle/Tests/Translation/TranslatorTest.php

```diff
@@ -255,8 +255,10 @@ public function testCachedCatalogueIsReDumpedWhenScannedDirectoriesChange()
                     __DIR__.'/../Fixtures/Resources/translations/messages.fr.yml',
                 ],
             ],
-            'scanned_directories' => [
-                __DIR__.'/../Fixtures/Resources/translations/',
+            'cache_vary' => [
+                'scanned_directories' => [
+                    '/Fixtures/Resources/translations/',
```

Comment:
I think we need to remove the leading `/` too, to avoid ambiguity between `/projectRoot/Fixtures/Resources/translations/` and `/Fixtures/Resources/translations/` (outside project root, left unchanged). Paths starting with a `/` here will look the same than absolute paths

Reply:
I added a ltrim.